tests/acceptance/testall

```diff
@@ -53,6 +53,11 @@ if [ -f /usr/xpg4/bin/grep ]; then
 fi
 
 #
+# Close stdin file descriptor to avoid tty_interactive check in cf-agent being success.
+#
+exec <&-
```

Comment:
Oh, that's true, I didn't think about that. Maybe skip it in that case. You'll have to do the closing further down then, after scanning arguments.